docker-compose.yml

```diff
@@ -11,6 +11,8 @@ services:
       - esdata:/usr/share/elasticsearch/data
     environment:
       - discovery.type=single-node
```

Comment:
My guess is, that once you are in a cluster setup, you have to also remove this line?

Reply:
```suggestion
      # Comment out the line below for single-node
      - discovery.type=single-node
```